deployment/monocular/templates/api-deployment.yaml

```diff
@@ -23,13 +23,13 @@ spec:
           httpGet:
             path: /healthz
             port: {{ .Values.api.service.internalPort }}
-          initialDelaySeconds: 60
+          initialDelaySeconds: 180
           timeoutSeconds: 10
         readinessProbe:
           httpGet:
             path: /healthz
             port: {{ .Values.api.service.internalPort }}
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
```

Comment:
I think we can leave this as-is as the readiness probe will just determine whether the pod is reachable, but won't kill it.